nmostesting/GenericTest.py

```diff
@@ -503,9 +503,9 @@ def do_test_api_resource(self, resource, response_code, api):
         # Test URLs which include a {resourceId} or similar parameter
         if resource[1]['params'] and len(resource[1]['params']) == 1:
             path = resource[0].split("{")[0].rstrip("/")
-            if path in self.saved_entities:
+            if path in self.saved_entities[api]:
```

Comment:
Is `api` guaranteed to have been added by this point? I assume so...

Would it be nice to make the same change to `omit_paths` though that will require changing the test suites that use it?

Reply:
> Is `api` guaranteed to have been added by this point?

Excellent question, I can be missing https://github.com/AMWA-TV/nmos-testing/pull/614/files#diff-251e8b81551664aae6c5685371b7531443584eba2e2f8d22f98a4f128dcc0eedR601-R602

```suggestion
            if api in self.saved_entities and path in self.saved_entities[api]:
```